Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -431,7 +457,7 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
       return .skipChildren
     }
 
-    if node.leftSquareBracket.presence == .missing && node.rightSquareBracket.presence == .present {
+    if node.leftSquareBracket.isMissing && node.rightSquareBracket.presence == .present {
```

Comment:
Personally I'm not a huge fan of introducing an accessor here. If you *really* want it then I'd also want a `isPresent`. Or if we've decided there's only ever going to be two values here, then changing the `.presence == .present` to `!isMissing` would be another option.